src/main/resources/META-INF/plugin.xml

```diff
@@ -9,7 +9,14 @@
 
     <change-notes>
         <![CDATA[
-          <li> The plugin cannot find the NPM executable in some cases.</li>
+          <li> Add 'Show in dependencies tree' button in the package descriptors.</li>
+          <li> Show license and top issue severity information when hovering above a dependency in the package descriptor.</li>
```

Comment:
We can shorten to:
Show license and top issue severity information when hovering above a dependency.